test/parallel_api/experimental/asynch-scan.pass.cpp

```diff
@@ -46,48 +46,48 @@ test_with_buffers()
         // transform inclusive (2 overloads)
         auto my_policy1 = TestUtils::make_device_policy<class Scan1>(my_policy);
         auto alpha = dpl::experimental::transform_inclusive_scan_async(my_policy1, dpl::begin(x), dpl::end(x), dpl::begin(y), std::plus<int>(), [](auto x) { return x * 10; });
-        auto result1 = __dpl_sycl::__get_host_access(alpha.get().get_buffer())[n-1];
+        auto result1 = TestUtils::get_host_access(alpha.get().get_buffer())[n-1];
```

Comment:
We have a lot of `get_host_access` calls in the tests. Are we able to rewrite this line as in other tests without new function in utilities?

Reply:
Thanks for the comment. I see that some tests use `dpl::copy` and some (overwhelming majority) use `get_host_access` directly. I have replaced these functions with `get_host_access`, which is available in the oldest supported compiler: Intel(R) oneAPI DPC++/C++ Compiler 2022.2.1` and newer.

Your comment also motivated me to check if there are other outdated wrappers. There were, I have removed them, contemporary functionality directly.

Reply:
ok